draft-ietf-quic-qlog-main-schema.md

```diff
@@ -157,6 +157,13 @@ logged as `float64` in the millisecond resolution.
 Other qlog documents can define their own CDDL-compatible (struct) types
 (e.g., separately for each Packet type that a protocol supports).
 
+The member fields in all qlog CDDL type definitions are listed without any
+expectation of a fixed ordering. Specific qlog serializations MAY impose a
+stricter ordering of fields, but the default JSON and JSON Text Sequences
+serializations discussed in this document (see {{concrete-formats}}) do not.
+qlog tools SHOULD NOT assume a specific ordering of qlog member field
+occurrences in general.
```

Comment:
I'm not sure this hits the right notes for me. An alternative suggestion is something like 

```suggestion
The ordering of member fields in qlog CDDL type definitions is not significant.
The ordering of member field in the serialization formats defined in this
document, JSON ({{format-json}}) and JSON Text Sequences ({{format-json-seq}}),
is not significant and qlog tools MUST NOT assume so. Other qlog serialization
formats MAY define field order significance, if they do they MUST define
requirements for qlog tools supporting those formats.
```

Reply:
Agree that's a much better version :) accepted